src/test/java/app/fitbuddy/jpa/service/crud/AppUserCrudServiceTest.java

```diff
@@ -141,4 +144,33 @@ void readByName_whenAppUserFound_shouldReturnAppUserDto() {
 			assertEquals(appUserDtoMock, actualAppUserDto);
 		}
 	}
+	@Nested
+	class ReadMany {
+		@Test
+		void readMany_whenNoUsersFound_shouldReturnEmptyList() {
+			when(appUserCrudRepository.findAll()).thenReturn(Collections.emptyList());
+
+			List<AppUserDto> actualAppUserDtos = instance.readMany();
+
+			assertEquals(0, actualAppUserDtos.size());
+		}
+
+		@Test
+		void readMany_whenUsersFound_shouldReturnListOfAppUserDtos() {
+			List<AppUser> appUsersMock = new ArrayList<>();
+			AppUser appUserMock = AppUserTestHelper.getMockAppUser(1, "name", "password");
+			appUsersMock.add(appUserMock);
+
+			List<AppUserDto> appUserDtosMock = new ArrayList<>();
+			AppUserDto appUserDtoMock = new AppUserDto(1, "name", "password", "roleName");
+			appUserDtosMock.add(appUserDtoMock);
```

Comment:
Of course this code works, but we could really use the `List.of()` Java feature to shorten this part. Additionally, `List.of()` returns an immutable List. As we won't need to modify the List, this act as a "safe guard" so we cannot modify the List by mistake. 👍 

```
AppUser appUserMock = AppUserTestHelper.getMockAppUser(1, "name", "password");
List<AppUser> appUsersMock = List.of(appUserMock);

AppUserDto appUserDtoMock = new AppUserDto(1, "name", "password", "roleName");
List<AppUserDto> appUserDtosMock = List.of(appUserDtoMock);
```

Then we can easily remove the unnecessary import too: `import java.util.ArrayList;`

Reply:
Sure ,will change that. Thanks for the information.